src/planscape/rscripts/forsys.R

```diff
@@ -351,10 +386,10 @@ get_min_project_area <- function(scenario) {
   }
 
   log_info(
-    paste0(
-      "Stand size ",
+    paste(
+      "Stand size",
       stand_size,
-      " chosen. Minimum project area is ",
+      "chosen. Minimum project area is",
```

Comment:
this is the nittiest of nits but looks like we do want and extra space here at the end